ops/helmfiles/dagster/migrate.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Migrates our dagster cluster DB schema
+# (taken from https://docs.dagster.io/deployment/guides/kubernetes/how-to-migrate-your-instance)
+
+function msg() {
+  echo $1 >&2
+}
+
+function scale_down() {
+  kubectl -n dagster scale deploy $1 --replicas=0
+}
+
+function scale_up() {
+  kubectl -n dagster scale deploy $1 --replicas=$2
```

Comment:
i was thinking you could also compute the deployment name/replica count here, maybe returning the replica count from the function with an `echo`.